spec/lib/pdf_filler/md502_su_pdf_spec.rb

```diff
@@ -6,67 +6,52 @@
   let(:submission) { create :efile_submission, tax_return: nil, data_source: intake }
   let(:pdf) { described_class.new(submission) }
 
+  before do
+    intake.direct_file_data.primary_ssn = primary_ssn
+    intake.direct_file_data.spouse_ssn = spouse_ssn
+    allow_any_instance_of(Efile::Md::Md502SuCalculator).to receive(:calculate_line_ab).and_return 100
+    allow_any_instance_of(Efile::Md::Md502SuCalculator).to receive(:calculate_line_u).and_return 100
+  end
+
   describe "#hash_for_pdf" do
     let(:file_path) { described_class.new(submission).output_file.path }
     let(:pdf_fields) { filled_in_values(file_path) }
 
+    let(:intake) do
+      create(
+        :state_file_md_intake,
+        primary_first_name: "Janet",
+        primary_middle_initial: "G",
+        primary_last_name: "Jormp",
+        spouse_first_name: "Jane",
+        spouse_middle_initial: "M",
+        spouse_last_name: "Jomp",
+        )
+    end
+    let(:spouse_ssn) { "987654321" }
+    let(:primary_ssn) { "345678901" }
     it 'uses field names that exist in the pdf' do
       missing_fields = pdf.hash_for_pdf.keys.map { |k| k.to_s.gsub("'", "&apos;").to_s } - pdf_fields.keys
       expect(missing_fields).to eq([])
     end
 
-    context 'without interest reports' do
```

Comment:
i removed this, because the presence of interest reports is tested elsewhere in the calculator, so there's no need to have these two tests especially if the only difference is the "ab" field.